lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart

```diff
@@ -234,7 +277,8 @@ mixin MailboxWidgetMixin {
       EdgeInsets? padding
     }
   ) {
-    return Padding(
+    return Container(
```

Comment:
Why use `color: Colors.transparent` ?

Reply:
By default it will take the child's color. I want it to be able to change color later

Reply:
Oh, i set to debug, :). now i delete it. Thanks